frontend/packages/helm-plugin/src/components/forms/HelmChartRepository/CreateHelmChartRepositoryForm.tsx

```diff
@@ -72,13 +72,21 @@ const CreateHelmChartRepositoryForm: React.FC<FormikProps<FormikValues> & {
 
   const formTitle =
     !showScopeType && formData.scope === ProjectHelmChartRepositoryModel.kind
-      ? t('helm-plugin~Create ProjectHelmChartRepository')
-      : t('helm-plugin~Create Helm Chart Repository');
+      ? t('helm-plugin~{{formType}} ProjectHelmChartRepository', {
+          formType: existingRepo ? 'Edit' : 'Create',
+        })
+      : t('helm-plugin~{{formType}} Helm Chart Repository', {
+          formType: existingRepo ? 'Edit' : 'Create',
+        });
```

Comment:
Same translation issue here. Edit and Create should not be variables. This should be part of the translation. Keep it simple:

```suggestion
      ? (
        existingRepo
          ? t('helm-plugin~Edit ProjectHelmChartRepository')
          : t('helm-plugin~Create ProjectHelmChartRepository')
      )
      : (
        existingRepo
          ? t('helm-plugin~Edit Helm Chart Repository')
          : t('helm-plugin~Create Helm Chart Repository')
      );
```